### ISSUE TITLE
Question/possible bug in casing

### ISSUE THREAD

**Issue by bunny-therapist** [quesurifn/yake-rust#11]
I don't know enough to say what is right or wrong here, but this crate is [here](https://github.com/quesurifn/yake-rust/blob/master/yake_rust/src/lib.rs#L268) doing
```
cand.casing /= 1.0 + cand.tf.ln_1p();
```
but the [same computation in the LIAAD implementation of yake](https://github.com/LIAAD/yake/blob/master/yake/datarepresentation.py#L377) reads:
```
self.WCase = max(self.tf_a, self.tf_n) / (1. + math.log(self.tf))
```
(WCase is what we call "casing", tf_n appears to be what we call tf_u, even though the way it is computed differs.)

In other words, we have 1+ln(1+tf) in the denominator but LIAAD has 1+ln(tf)
We differ by a +1 in the logarithm since we are using `ln_1p` instead of `ln`.

Is this intentional? Is this correct?

**Comment by bunny-therapist** [comment]
Yeah, this is definitely a bug. Fixing it brings a lot of examples closer to LIAAD/yake.

@xamgore
